common/log/testlogger/testlogger.go

```diff
@@ -34,11 +31,6 @@ import (
 
 // New is a helper to create new development logger in unit test
 func New(t zaptest.TestingT) log.Logger {
-	if testing.Verbose() {
-		logger, err := loggerimpl.NewDevelopment()
-		require.NoError(t, err)
-		return logger
-	}
```

Comment:
I have no idea why anyone ever thought this was a good idea, so I removed it.

Since CI runs tests verbosely this _might_ lead to more race failures in tests, as it touches `*testing.T` more than it did before, and there are quite a lot of async logging goroutines that are not cleanly shut down across this repo.

**If** that happens and proves onerous to fix or work around, we could undo this.  So far I haven't seen other tests fail due to this change though.